docs/connector-development/connector-builder-ui/pagination.md

```diff
@@ -250,8 +250,6 @@ This can be done in a couple different ways:
 1. If you want to access fields on the records that you have defined through the record selector, you can use the `{{ last_records }}` object; so accessing the `key` field of the last record would look like `{{ last_records[-1]['key'] }}`. The `[-1]` syntax points to the last item in that `last_records` array.
 2. If you want to instead access a field on the raw API response body (e.g. your record selector filtered out the field you need), then you can use the `{{ response }}` object; so accessing the `key` field of the last item would look like `{{ response['data']['items'][-1]['key'] }}`.
 
-This API also has a boolean `has_more` property included in the response to indicate if there are more items to be retrieved, so the stop condition in this case should be `{{ response.data.has_more is false }}`
```

Comment:
Why remove this? This is necessary in this example because the user will need to use a Custom next page cursor

Reply:
I mentioned this up in the regular description when explaining the "custom" cursor value, but you are right that it should be noted together with the example. I added it back here and changed the example above